src/pages/blog/articles/_all-articles.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import ArticleCard from './_article-card'
+import FeaturedArticle from './_featured-article'
+import { Container, Flex } from 'components/containers'
+import { Text, LocalizedLinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import RightArrow from 'images/svg/black-right-arrow.svg'
+
+const VideoGrid = styled.div`
+    display: grid;
+    width: 100%;
+    height: 100%;
+    margin: 40px 0;
+    grid-template-columns: repeat(auto-fit, minmax(288px, 384px));
+    grid-row-gap: 40px;
+    grid-column-gap: 24px;
+    grid-template-rows: auto;
+`
+
+const AllArticles = ({ article_data }) => {
+    return (
+        <Container fd="column">
```

Comment:
Can add `width: 100%;padding: 0 16px;` for `tabletL` size for `Container` to follow the design padding.